test/BIP39Wallet.Tests.cs

```diff
@@ -13,8 +13,37 @@ public void CreateWallet_test()
         {
             // ReSharper disable once RedundantArgumentDefaultValue
             var accountInfo = new AElfWalletFactory().Create();
+            accountInfo.PrivateKey.Dispose();
             Assert.NotNull(accountInfo);
         }
+        [Fact]
+        public void CreateWallet_test_support_multi_language()
+        {
+            var accountInfo1 = new AElfWalletFactory().Create(language: Language.ChineseSimplified);
+            Assert.NotNull(accountInfo1);
```

Comment:
a very minor thought: can it looks like 7 of these can be put into an array and iterated over maybe? tho dunno if that makes it less readable 🤔 
```
Language[] languages = new Language[] { Language.ChineseSimplified,  Language.ChineseTraditional ,....};
var accountInfo;
foreach (Language language in languages)
{
             accountInfo = new AElfWalletFactory().Create(language: Language.ChineseTraditional);
            Assert.NotNull(accountInfo); //this should break the loop if an assert-fails iirc
}
```

Reply:
plz check the latest version of code, https://github.com/xiboXD/BIP39Wallet-NBitcoin/blob/dev/test/BIP39Wallet.Tests.cs, Theory method of xUnit is used for this function